deluge/tests/test_tracker_icons.py

```diff
@@ -45,6 +44,15 @@ def test_get_google_ico(self):
         d.addCallback(self.assertEqual, icon)
         return d
 
+    def test_get_google_ico_hebrew(self):
+        # Google.co.il page must be read as UTF-8
+        # So we'll test whether it can be read
```

Comment:
I fixed this when merging but it's better to use a docstring when describing the test (I know others tests are using a comment here)